examples/angular_bazel_architect/src/app/app-routing.module.ts

```diff
@@ -4,5 +4,5 @@ import {RouterModule, Routes} from '@angular/router';
 
 const routes: Routes = [];
 
-@NgModule({imports: [RouterModule.forRoot(routes)], exports: [RouterModule]})
+@NgModule({imports: [RouterModule.forRoot(routes, { relativeLinkResolution: 'legacy' })], exports: [RouterModule]})
```

Comment:
I think this should be reverted as there are no routes affected by the related bug IMO.

Reply:
@saxicek, agreed. Mind opening a PR? thanks.

Reply:
Ok, opened https://github.com/bazelbuild/rules_nodejs/pull/2530.